tests/unit/environments/test_environments_builders.py

```diff
@@ -271,3 +271,63 @@ def test_build_environment_api_key_model():
 
     # Then
     assert environment_key_model.key == environment_key_dict["key"]
+
+
+def test_build_environment_model_with_deprecated_field():
```

Comment:
Sorry for being a pedant, but the more `-> None` annotations are added now, the less we'll suffer when enforcing strict typing later

Reply:
Good catch, sorry! Will add now!

Reply:
(and _never_ apologise for pedantry!)